### PR TITLE
[skwasm] Don't mark the backend texture as mipmapped, since it isn't.

### PR DESCRIPTION
**Pull request by eyebrowsoffire**
We don't actually produce mipmaps for any of the textures generated through the `TextureSourceImageGenerator`, even if it is requested of us. So we shouldn't mark the texture as mipmapped in this case. This will allow rendering mechanisms to fall back to a non-mipmapped rendering strategy as needed.

### CODE REVIEW

**Review by yjbanov — APPROVED** [review]
lgtm

File: lib/web_ui/skwasm/image.cpp [hunk 1]
```diff
@@ -100,7 +100,7 @@ class TextureSourceImageGenerator : public GrExternalTextureGenerator {
     glInfo.fTarget = GL_TEXTURE_2D;
```
Comment (kjlubick) [inline lib/web_ui/skwasm/image.cpp:100]: nit: also remove mipmapped on line 94 to make it more clear that param is not used in this impl

**Review by kjlubick — COMMENTED** [review]
LGTM with nit

File: lib/web_ui/skwasm/image.cpp (hunk 1 above)
Comment (eyebrowsoffire) [reply lib/web_ui/skwasm/image.cpp:100]: Ah, good idea. Unfortunately this already merged. I'll see about cleaning this up in a followup.

**Pull request merged**